ci-operator/step-registry/fips-check/node-scan/fips-check-node-scan-commands.sh

```diff
@@ -76,7 +76,7 @@ oc --request-timeout=300s -n "$project" debug node/"$master_node_0" -- chroot /h
 out=$(oc --request-timeout=300s -n "$project" debug node/"$master_node_0" -- chroot /host bash -c "cat /$report" || true)
 echo "The report is: $out"
 oc delete ns $project || true
-res=$(echo $out | grep -E 'Failure Report|Successful run with warnings|Warning Report')
+res=$(echo $out | grep -E 'Failure Report|Successful run with warnings|Warning Report' || true)
```

Comment:
> No, it is the special characters which make the junit report not readable.

Hmm, half right. If you check `The result is: ...` line from /artifacts/azure-stack-ipi-proxy-fips-f14/fips-check-node-scan/build-log.txt of the link pasted by your PR's starting description comment, you see one single line containing long content:
```
sha256:8c00... Writing ... signatures I0102 10:26:38.581637 1 main.go:278] ... I0102 10:26:38.582249 1 types_config.go:12]...&{Components:[] ... rpm -ql ....el9.noarch ---- Failure Report +---...+ | RPM NAME
```
This is why the whole content is assigned to $res.
Fix it by adding double quotes on $out:
```suggestion
res=$(echo "$out" | grep -E 'Failure Report|Successful run with warnings|Warning Report' || true)
```